app/main.py

```diff
@@ -1,2 +1,63 @@
+from dataclasses import dataclass
+from typing import Hashable, Any
+
+
+@dataclass
+class Node:
+    key: Hashable
+    value: any
```

Comment:
The type for `value` should be `Any` instead of `any`. This is a minor issue but should be corrected for consistency with Python's typing module.